presto-docs/src/main/sphinx/release/release-0.233.1.rst

```diff
@@ -0,0 +1,8 @@
+===============
+Release 0.233.1
+===============
+
+General Changes
+_______________
+* Fix an issue where queries with both ``COALESCE`` and ``FULL OUTER JOIN`` may produce incorrect
```

Comment:
@kaikalur  was related to bucketing, I believe this was from @yingsu00

Reply:
> @kaikalur was related to bucketing, I believe this was from @yingsu00

== RELEASE NOTES ==
Fixed bucket pruning when the predicate is IS NULL and the table actually has rows with that value which we were previously not considering.

Reply:
@kaikalur Gotcha. Can you rephrase this to be user-facing. User do not understand what is "buck pruning".

Reply:
Bucketing is a Hive fix, so this should go to `Hive changes`, right?

`Fix an issue where queries with the predicate `IS NULL` on bucketed columns would always return 0 results.` ?

Reply:
Fix incorrect results returned when selecting from a bucketed table when bucket predicate is NULL